majasdarbs/src/main/java/io/codelex/kontrolDarbs/exercise3/Joiner.java

```diff
@@ -0,0 +1,22 @@
+package io.codelex.kontrolDarbs.exercise3;
+
+import java.util.function.Function;
+
+public class Joiner<T> {
```

Comment:
Ši klase ir uzrakstīta par sarežģītu - te funkcijas nebija jāpielieto.
Kā arī pārlasi uzdevumu:
`Joiner<T> class, which in the constructor will take a separator (string) and have a join() method that allows you to specify any number of T-type objects`

Viņam būtu jāstrādā, piemēram šadi:
Joiner<Intger> joiner = new Joiner<>("x");
String result1 = joiner.join(1,2,3);
String result2 = joiner.join(1,2,3,4,5);
//result1 būtu jābūt 1x2x3
//result2 būtu jābūt 1x2x3x4x5